enarx-keep/tests/integration_tests.rs

```diff
@@ -111,5 +122,15 @@ fn write_stderr() {
 
 #[test]
 fn write_emsgsize() {
-    run_test("write_emsgsize", 0);
+    run_test("write_emsgsize", 0, None);
+}
+
+#[test]
+fn read() {
+    let input = "hello world\n";
+    let child = run_test("read", 0, Some(input));
+    let stdout = child.stdout.unwrap();
+
+    let buf: [u8; 12] = read_item(stdout).unwrap();
```

Comment:
```suggestion
    let buf: [u8; INPUT.len()] = read_item(stdout).unwrap();
```

I wish this was possible... :(

Reply:
I think it worked.